requirements.txt

```diff
@@ -5,6 +5,7 @@ SQLAlchemy>=1.3.12
 python-jose>=3.2.0
 passlib[bcrypt]>=1.7.4
 python-multipart==0.0.5
+boto3
```

Comment:
for now, we don't need it?

Reply:
It's is used by the S3 demonstrator. I would keep it and remove it later hen we won't rely on s3 anymore.